[2021-06-25 09:45:14,374] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_3 2020-01-09T00:00:00+00:00 [queued]>
[2021-06-25 09:45:14,662] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_3 2020-01-09T00:00:00+00:00 [queued]>
[2021-06-25 09:45:14,673] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-06-25 09:45:14,709] {taskinstance.py:1068} INFO - Starting attempt 2 of 2
[2021-06-25 09:45:14,736] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-06-25 09:45:14,792] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): task_3> on 2020-01-09T00:00:00+00:00
[2021-06-25 09:45:14,822] {standard_task_runner.py:52} INFO - Started process 18921 to run task
[2021-06-25 09:45:14,930] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_3', '2020-01-09T00:00:00+00:00', '--job-id', '393', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmpo661w5a0', '--error-file', '/tmp/tmprevww0m9']
[2021-06-25 09:45:14,968] {standard_task_runner.py:77} INFO - Job 393: Subtask task_3
[2021-06-25 09:45:16,804] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag.task_3 2020-01-09T00:00:00+00:00 [running]> on host airflowvm
[2021-06-25 09:45:19,670] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_3
AIRFLOW_CTX_EXECUTION_DATE=2020-01-09T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-01-09T00:00:00+00:00
[2021-06-25 09:45:19,789] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-06-25 09:45:19,803] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'sleep 3']
[2021-06-25 09:45:20,090] {subprocess.py:75} INFO - Output:
[2021-06-25 09:45:23,187] {subprocess.py:83} INFO - Command exited with return code 0
[2021-06-25 09:45:23,585] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_3, execution_date=20200109T000000, start_date=20210625T094514, end_date=20210625T094523
[2021-06-25 09:45:23,935] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-06-25 09:45:24,008] {local_task_job.py:151} INFO - Task exited with return code 0
